crates/rolldown/tests/rolldown/tree_shaking/pure_annotation/artifacts.snap

```diff
@@ -18,7 +18,7 @@ const c = test();
 //#endregion
 //#region d.js
 function test2() {}
-const e = /* #__PURE__ */ test2();
```

Comment:
This is expected, 
the original code is 

```js
function test2() {}

export const d = /* #__PURE__ */                 test2(), e =  test2();
              
```

`PURE` should attached to the first `test2()`.